docs/languages.md

```diff
@@ -0,0 +1,114 @@
+## Supported languages for the Original Language field
+
+The following table shows all currently supported languages (BCP47 format) for the `originalLanguage` field and their corresponding SAP2 language code.
+
```

Comment:
Thanks for adding all languages. Shall we add the information this is based on SAP Note https://launchpad.support.sap.com/#/notes/73606?

Something like this?

```suggestion
The list of supported languages is based on [SAP Note 73606 - Supported Languages and Code Pages](https://launchpad.support.sap.com/#/notes/73606).

```

Reply:
sounds good 👍